extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -88,7 +88,7 @@
   display: flex !important;
   flex-direction: row !important;
   justify-content: flex-end !important;
-  padding-top: 6px !important;
+  padding: 8px 4px !important;
```

Comment:
The thoughts on mobile is that many platforms (in apps and through AmpStoryPlayer) will want to add border radiuses to the viewers, which is why we add the margins in mobile view as well (for example the Discover feed has small radiuses now because it would hide the progress bar otherwise). We thought about being able to toggle this through a viewer message or config, but it would be more inconsistent overall, especially if viewers don't configure things properly (most of the times people don't read every sentence in the docs).

So yes, it's intended to have the added margins in mobile as well, so viewers can use larger border radiuses (which looks more modern).